.github/workflows/keep-alive.yml

```
@@ -0,0 +1,12 @@
+name: keep alive
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

I think running this once a week should be enough?

```suggestion
    - cron: "0 0 * * 0"
```

This would be on Sundays.